pkg/client/restclient/config.go

```diff
@@ -152,6 +157,22 @@ type ContentConfig struct {
 	NegotiatedSerializer runtime.NegotiatedSerializer
 }
 
+// Returns Host and AlternateHosts as a single slice, without duplicates
```

Comment:
Comment is not quite correct, does not prevent duplicates in AlternateHosts, only prevents Host from appearing twice.